objects/official/ride/official.rmct2/object.json

```diff
@@ -0,0 +1,3470 @@
+{
+    "id": "official.rmct2",
+    "authors": ["spacek531"],
+    "version": "1.0",
+    "originalId": "00000000|RMCT2   |00003000",
```

Comment:
This is not correct. It should consist of the DAT header and checksum of the original .DAT object if there is one. If not, let me know and I´ll calculate the values.

Reply:
This is the actual header of the real DAT